config/initializers/foreman.rb

```diff
@@ -28,10 +28,12 @@
 Dashboard::Loader.load
 
 # clear our users topbar cache
-begin
-  User.unscoped.pluck(:id).each do |id|
-    Rails.cache.delete("views/tabs_and_title_records-#{id}")
+if defined?(User)
```

Comment:
That feels limiting given we are trying to access a model in an initializer and there are states of the application where that model or its tables may or may not exist yet.